core/data_migration_scripts/EE_DMS_Core_4_1_0.dms.php

```diff
@@ -1452,7 +1452,7 @@ public function convert_image_url_to_attachment_and_attach_to_post(
                 update_post_meta($new_cpt_id, '_thumbnail_id', $attachment_post_id);
             } else {
                 $migration_stage->add_error(sprintf(esc_html__(
-                    "Could not update event image %s for CPT with ID %d, but attachments post ID is %d",
+                    "Could NOT update event image %s for CPT with ID %d, but attachments post ID is %d",
```

Comment:
Leave lowercase so it doesn't break string translations?

Reply:
also